native_client/dotnet/DeepSpeechClient/Extensions/NativeExtensions.cs

```diff
@@ -56,7 +56,7 @@ private static Models.CandidateTranscript PtrToCandidateTranscript(this IntPtr i
             managedTranscript.Confidence = transcript.confidence;
 
             //we need to manually read each item from the native ptr using its size
-            var sizeOfTokenMetadata = Marshal.SizeOf(typeof(TokenMetadata));
+            var sizeOfTokenMetadata = Marshal.SizeOf<TokenMetadata>();
```

Comment:
I defer to @carlfm01 for that kind of changes

Reply:
Ah nice, generics, this change is correct.